README.md

```diff
@@ -74,10 +75,10 @@ Select 1:
 
 Identify the three places CSS can go, and rank them in terms of specificity:
 
-```text
-1.
-2.
-3.
+```I think this is what is being asked:
+1.Inline coding i.e. <style></style> within the HTML Doc
+2.Separate CSS file linked to the HTML Doc.
+3.Element Names
 ```
```

Comment:
Not Quite... 

1 Inline but within the HTML element itself  ` style=`
2 Within `<style></style>` tags in the `<head>` of an html document
3 In an external or "separate css" file or stylesheet